virttest/libvirt_vm.py

```diff
@@ -2449,6 +2449,24 @@ def get_interface_mac(self, interface):
         session.close()
         return mac.strip()
 
+    def get_distro(self):
+        """
+        Get distribution name of the vm instance.
+        """
+        distro = ""
+        session = self.wait_for_login()
+        cmd = "cat /etc/os-release | grep '^ID='"
+        try:
+            status, output = session.cmd_status_output(cmd, timeout=300)
+            if status:
+                raise virt_vm.VMError("Unable to get the distro name: %s"
+                                      % output)
+            else:
+                distro = output.split('=')[1].strip()
+        finally:
+            session.close()
+            return distro
+
```

Comment:
@will-Do platform.dist() gives distro name of host but we need to check the distro of guest by using session.cmd_status_output. so I thought this will be fine.